coord/src/rmr/rmr.c

```diff
@@ -545,10 +546,22 @@ int MRIteratorCallback_ResendCommand(MRIteratorCallbackCtx *ctx, MRCommand *cmd)
                                ctx);
 }
 
+// Use after modifying `pending` (or any other variable of the iterator) to make sure it's visible to other threads
+void MRIteratorCallback_ProcessDone(MRIteratorCallbackCtx *ctx) {
+  __atomic_fetch_sub(&ctx->ic->inProcess, 1, __ATOMIC_RELEASE);
```

Comment:
Why it needs to be atomic? don't we always decrease it/set it from a single thread?

Reply:
Currently, a single thread sets (with no race), and then a single one decreases while another one observes the change. the atomicity is used more for memory fencing than for counter-modification.

Reply:
So why not use volatile?